src/viking.js

```diff
@@ -1,14 +1,72 @@
 // Soldier
-class Soldier {}
+class Soldier {
+  constructor(health, strength) {
+    this.strength = 150;
+    this.health = 300;
+  }
```

Comment:
Hey Marten, hier schreibst du einfach statt der Zahlen, strength und health rein, weil es immer unterschiedlich sein soll, also keine feste Summe für alle.
this.health = health usw